.openpublishing.redirection.json

```
@@ -1,1234 +1,1184 @@
+        },
+        {
+            "source_path": "docs/develop/resource-limits-and-performance-optimization.md",
+            "redirect_url": "/office/dev/add-ins/concepts/resource-limits-and-performance-optimization",
+            "redirect_document_id": true
+        },
+        {
+            "source_path": "docs/develop/define-add-in-commands.md",
+            "redirect_url": "/office/dev/add-ins/develop/create-addin-commands",
+            "redirect_document_id": false
+        },
+        {
+            "source_path": "docs/excel/excel-add-ins-extend-excel.md",
+            "redirect_url": "/office/dev/add-ins/excel/excel-add-ins-overview",
+            "redirect_document_id": true
+        },
+        {
+            "source_path": "docs/excel/custom-functions-shared-overview.md",
+            "redirect_url": "/office/dev/add-ins/develop/configure-your-add-in-to-use-a-shared-runtime",
+            "redirect_document_id": false
+        },
+        {
+            "source_path": "docs/excel/configure-your-add-in-to-use-a-shared-runtime.md",
+            "redirect_url": "/office/dev/add-ins/develop/configure-your-add-in-to-use-a-shared-runtime",
+            "redirect_document_id": true
+        },
+        {
+            "source_path": "docs/excel/run-code-on-document-open.md",
+            "redirect_url": "/office/dev/add-ins/develop/run-code-on-document-open",
+            "redirect_document_id": true
+        },
+        {
+            "source_path": "docs/excel/show-hide-add-in.md",
+            "redirect_url": "/office/dev/add-ins/develop/show-hide-add-in",
+            "redirect_document_id": true
+        },
+        {
+            "source_path": "docs/excel/excel-add-ins-excel-add-ins-advanced-concepts.md",
+            "redirect_url": "/office/dev/add-ins/excel/excel-add-ins-core-concepts",
+            "redirect_document_id": true
+        },
+        {
+            "source_path": "docs/access/access-web-app-add-ins.md",
+            "redirect_url": "https://powerapps.microsoft.com",
+            "redirect_document_id": false
+        },
+        {
+            "source_path": "docs/develop/implementation-details.md",
+            "redirect_url": "/office/dev/add-ins/develop/understanding-the-javascript-api-for-office",
+            "redirect_document_id": false
+        },
+        {
+            "source_path": "docs/develop/office-js-versioning.md",
+            "redirect_url": "/office/dev/add-ins/develop/understanding-the-javascript-api-for-office",
+            "redirect_document_id": false
+        },
+        {
+            "source_path": "docs/develop/office-versions.md",
+            "redirect_url": "/office/dev/add-ins/develop/understanding-the-javascript-api-for-office",
+            "redirect_document_id": false
+        },
+        {
+            "source_path": "docs/develop/authentication-pnp.md",
+            "redirect_url": "/office/dev/add-ins/overview/office-add-in-code-samples",
```

You might be able to redirect to the auth section
```suggestion
            "redirect_url": "/office/dev/add-ins/overview/office-add-in-code-samples#authentication-authorization-and-single-sign-on-sso",
```